source/addComment.js

```diff
@@ -1,3 +1,32 @@
-$(document).ready(function(){
-  // add your code here
-})
+/*jshint esversion: 6 */
+
+$('button').on('click', function() {
+  $('button').hide();
```

Comment:
Remember that in bigger apps or more complex pages, this would call this logic on *any* button click.